packages/ui/core-components/src/lib/unsorted/ui/Card.svelte

```diff
@@ -0,0 +1,14 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script>
+	export let className = '';
+</script>
+
+<div
+	class={`border border-base-400 rounded-md px-3 py-1 shadow-base-200 ${className}`}
```

Comment:
I think className needs to go before the other props or you will not be able to override them

eg if you want to change to `rounded-sm` it needs to be before the `rounded-md` default

Reply:
Just putting the classes first isn't 100% reliable afaik. There's a helper function `cn` in $lib/utils that does tailwind merges. 

```
import { cn } from $lib/utils 

... 

class={cn(className, 'border border-base-400...)}
```